### PR TITLE
PE-4752: Wallet address in login page PoC

### PR DESCRIPTION
**Pull request by matibat**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/1l2lgcfblkooo

### CODE REVIEW

File: lib/authentication/login/blocs/login_bloc.dart [hunk 1]
```diff
@@ -34,11 +37,21 @@ class LoginBloc extends Bloc<LoginEvent, LoginState> {
   @visibleForTesting
   ProfileType? profileType;
 
+  Future<String?> getWalletAddress() async {
```
Comment (thiagocarvalhodev) [inline lib/authentication/login/blocs/login_bloc.dart:40]: Can we implement this method on the `ArDriveAuth` class?

File: lib/user/repositories/user_repository.dart [hunk 2]
```diff
@@ -14,6 +14,7 @@ abstract class UserRepository {
     Wallet wallet,
   );
   Future<void> deleteUser();
+  Future<String?> getOwnerOfDefaultProfile();
```
Comment (thiagocarvalhodev) [inline lib/user/repositories/user_repository.dart:17]: Can you add a comment here explaining why we can receive a null value? e.g. returns null if no profile is available

**Review by thiagocarvalhodev — CHANGES_REQUESTED** [review]
Left a few changes that you can consider implementing

File: lib/authentication/login/blocs/login_bloc.dart (hunk 1 above)
Comment (matibat) [reply lib/authentication/login/blocs/login_bloc.dart]: There we go - 7f293297c9285c708b5fdf29c9ed357c1d6f46c6

File: lib/user/repositories/user_repository.dart (hunk 2 above)
Comment (matibat) [reply lib/user/repositories/user_repository.dart:17]: Here it is - 756f7141bbff1adaa57dc4d6ebfa456d8449b3af

**Review by thiagocarvalhodev — CHANGES_REQUESTED** [review]
Thanks for applying the changes! We are just missing the tests on the new method of the repository. I can approve after that

**Review by thiagocarvalhodev — DISMISSED** [review]

**Review by thiagocarvalhodev — APPROVED** [review]

**Pull request merged**